test/Dataflow/demanded-bits6.opt

```diff
@@ -1,8 +1,11 @@
 
 
+; RUN: %souper-check -souper-use-alive %s > %t 2>&1
+; RUN: %FileCheck %s < %t
 ; RUN: %souper-check %s > %t 2>&1
 ; RUN: %FileCheck %s < %t
 
+
```

Comment:
look for an eliminate spurious white space changes in your patch, there are a few others

Reply:
fixed